source/backend/routes/recipes.js

```diff
@@ -28,6 +29,30 @@ router.get('/recipeId/:recipeId', async (req, res) => {
   }
 });
 
+/* PUT /recipes/:recipeId */
+router.put('/:recipeId', verifyUserToken, async (req, res) => {
+  try {
+    const { userId } = req.userInfo;
+    const { recipeId } = req.params;
+
+    const recipe = await recipesModel.getByUserIdAndRecipeId(userId, recipeId);
+    if (recipe.length == 0) {
+      return res.status(401).json({msg: 'Unauthorized to edit recipe'});
+    }
+
+    const updatedRecipe = req.body;
+    await recipesModel.updateByRecipeId(updatedRecipe, recipeId);
```

Comment:
I couldnt find the updateByRecipeId function. did you mean updateRecipe()?

Reply:
You can change the updateRecipe in model to updateByRecipeId

Reply:
Yes, I forgot to change the function name accordingly. I will call updateRecipe instead. Thanks for pointing that out.